integrade/tests/ui/test_login.py

```diff
@@ -13,16 +13,15 @@
 
 import pytest
 
-from selenium.common.exceptions import NoSuchElementException, TimeoutException
-# from selenium.webdriver.common.by import By
+from selenium.webdriver.common.by import By
 from selenium.webdriver.support.ui import WebDriverWait
 
 from widgetastic.browser import Browser
 
 from integrade.config import get_config
 from integrade.utils import base_url
 
-from .utils import find_element_by_text, \
+from .utils import find_element_by_text, wait_for_input_value, \
```

Comment:
Usually we use parenthesis when import items does not fit on a line. But using `\` is OK too.